images/installer/Dockerfile

```diff
@@ -12,23 +12,54 @@ LABEL name="openshift/origin-ansible" \
       io.k8s.display-name="openshift-ansible" \
       io.k8s.description="A containerized openshift-ansible image to let you run playbooks to install, upgrade, maintain and check an OpenShift cluster" \
       io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible"
+      io.openshift.tags="openshift,install,upgrade,ansible" \
+      vcs-url="https://github.com/openshift/openshift-ansible" \
+      vcs-type="git" \
+      version="alpha"
 
 USER root
 
+# ansible and pip are in EPEL
+RUN yum install -y epel-release && yum clean all -y
+
+# workaround for https://github.com/openshift/openshift-ansible/issues/3111
+# install ansible via pip to lock version
```

Comment:
this is resolved with later ansible, should be able to use latest ansible from EPEL or origin repos